docs/src/concepts/toolchains.md

```diff
@@ -0,0 +1,58 @@
+# Toolchains
+
+Many `fuelup` commands deal with _toolchains_, a single installation of the
+Sway toolchain. `fuelup` supports **two** types of toolchains.
+
+1. Official toolchains which track the official release [channels] (only _latest_ for now);
+2. Custom toolchains and install individual components in a modular manner.
+
+[channels]: channels/index.md
+
+## Toolchain specification
+
+Standard release channel toolchain names have the following form:
+
+```
```

Comment:
This is also caught by the linter with the following reference, I think this is not a language (not sure though 😄 ) maybe we can place a `text` next to it.

```
MD040/fenced-code-language: Fenced code blocks should have a language specified
```
[Cool reference 2](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md040---fenced-code-blocks-should-have-a-language-specified)

```suggestion
```text
```